dbt/models/qc/schema.yml

```diff
@@ -486,3 +486,155 @@ models:
+    config:
+      tags:
+        - test_qc_sales
+    columns:
+      - name: buyer
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_iasworld_sales_null_values_buyer_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: deed_type
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_iasworld_sales_null_values_deed_type_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: price
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_iasworld_sales_null_values_price_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: seller
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_iasworld_sales_null_values_seller_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+
+  - name: qc.vw_sale_mydec_null_values
+    description: '{{ doc("view_vw_sale_mydec_null_values") }}'
+    config:
+      tags:
+        - test_qc_sales
+    columns:
+      - name: address
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_address_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: buyer
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_buyer_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: deed_type
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_deed_type_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: price
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_price_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: seller
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_seller_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+
+  - name: qc.vw_iasworld_sale_rowcount_matches_sales_mydec
+    description: '{{ doc("view_vw_iasworld_sale_rowcount_matches_sales_mydec") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_rowcount_matches_sales_mydec_check
+          expression: comparison != 'No significant difference'
+
+  - name: qc.vw_sale_mydec_change_in_high_low_value_sales
+    description: '{{ doc("view_vw_sale_mydec_change_in_high_low_value_sales") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_sale_mydec_change_in_high_low_value_sales_low_value_gt_25%
```

Comment:
**issue:** Let's convert the `%` here to `_pct` or similar, since I'm not sure how the `%` symbol will play with saving test outputs to Athena.